packages/storybook/stories/components/Spinner/documentation.mdx

```diff
@@ -0,0 +1,30 @@
+import { Meta } from '@storybook/blocks';
+import OverviewSpinner from '@ovhcloud/ods-components/src/components/spinner/documentation/overview-spinner.mdx';
+import SpecificationsSpinner from '@ovhcloud/ods-components/src/components/spinner/documentation/specifications/specifications-spinner.mdx';
+import StyleCustomizationSpinner from '@ovhcloud/ods-components/src/components/spinner/documentation/style-customization-spinner.mdx';
+import Usage from '@ovhcloud/ods-components/src/components/spinner/documentation/usage-guidelines/usage.mdx';
+import * as SpinnerStories from './spinner.stories';
+
+<Meta of={SpinnerStories} name='Documentation' />
+
+<div style={{backgroundColor: 'var(--ods-color-primary-500)', margin: '-1.5rem -1.5rem 3rem', minHeight: '5rem', position: 'relative', borderRadius: '0 0 1rem 1rem', padding: '1.25rem'}}>
```

Comment:
can we use a class from the assets css files?

Reply:
Now we can import Banner and add it in our component documentation to generate this banner.